ethereum/utils.py

```diff
@@ -428,17 +426,6 @@ def __init__(self):
 trie_root = Binary.fixed_length(32, allow_empty=True)
 
 
-class bcolors:
```

Comment:
It is only used by the `slogging` module so I moved it over to there.  Happy to move it back if it's a problem.